recipes/jeepney/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jeepney" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "a6f2aa72e61660248d4d524dfccb6405f17c693b69af5d60dd7f2bab807d907e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: flit install --deps none
+
+requirements:
+  build:
+    - python
+    - flit
+  run:
+    - python
+    - testpath
```

Comment:
`testpath` is only a test dependency, I think.

Reply:
Probably best to move it to test requires.

Reply:
Since we're only `import`ing here, not much point to that.